instagram-app/src/App.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import './App.css';
+import SearchBar from './components/SearchBar/SearchBar';
+import PostContainer from './components/PostContainer/PostContainer/PostContainer';
+import dummyData from './dummy-data';
+
+class App extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data : [],
+    }
+  }
+
+  componentDidMount() {
+    this.setState({ data: dummyData });
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <SearchBar />
+        <PostContainer post={this.state.data}/>
```

Comment:
I would have named this prop here posts instead of posts because post implies a single posts. It's a semantic thing.